www/_posts/2024/2024-01-22-outstanding-award.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post.html
+title: "Announcing: The Outstanding PyLady Award"
+tags: [PyLadies, Global, Community, Outreach]
+author: Lynn Root
+author_link: https://mastodon.online/@roguelynn
+---
+
+Introducing: the Outstanding PyLady Award!
+
+**TL;DR:** Know of a PyLady that should be recognized for all their work for Python and the community?
+[Nominate them (or yourself)][nomination form]!
+
+The PyLadies community is made up of many remarkable members and volunteers that make significant 
+contributions to bettering the Python community. The Global Council's [Outstanding PyLady award][award docs] is 
```

Comment:
This might have been an oversight. It is the "Outstanding PyLady award" hence a community award, not a GC one we should perhaps push a follow PR to avoid confusion

Reply:
Thank you for the clarification. Yes please update the text by creating a PR, and I believe the wording should be updated on kit.pyladies.com and the Google Form as well.